azkaban-webserver/build.gradle

```diff
@@ -119,56 +108,39 @@ dustjs {
   dest = 'build/dust'
 }
 
-task createDirs() << {
-  file('build/package/extlib').mkdirs()
-  file('build/package/plugins').mkdirs()
-}
+installDist.dependsOn 'lesscss'
 
-task copyWeb(type: Copy) {
-  from('src/web')
-  into('build/package/web')
-}
-
-task copyLess(type: Copy, dependsOn: ['lesscss', 'copyWeb']) {
-  from('build/less')
-  into('build/package/web/css')
-}
-
-task copyDust(type: Copy, dependsOn: ['dustjs', 'copyWeb']) {
-  from('build/dust')
-  into('build/package/web/js')
-}
-
-task copyJSLibs(type: Copy, dependsOn: ['bowerInstall', 'dustjs', 'copyWeb']) {
-  from('bower/')
-  into('build/package/web/js')
-}
-
-task copyDeps(type: Copy, dependsOn: 'build') {
-  from(project(':azkaban-common').configurations.compile)
-  from(configurations.compile)
-  into('build/package/lib')
-}
-
-task copyLibs(type: Copy, dependsOn: 'build') {
-  from('build/libs')
-  into('build/package/lib')
-}
-
-task copyPackage(type: Copy) {
-  from('src/package')
-  into('build/package')
-}
-
-task copy(dependsOn: [
-        'createVersionFile',
-        'createDirs',
-        'copyLess',
-        'copyDust',
-        'copyDeps',
-        'copyJSLibs',
-        'copyLibs',
-        'copyPackage']) {
+distributions {
+  main {
+    contents {
+      from('src/main/bash') {
+        into 'bin'
+        fileMode = 0755
+      }
+      from ('src/main/resources/conf') {
+        into 'conf'
+      }
+      from(configurations.runtime) {
+        into 'lib'
+      }
+      from(jar) {
+        into 'lib'
+      }
+      from ('src/web') {
+        into 'web'
+      }
+      from (lesscss.dest) {
+        into 'web/css'
+      }
+      from (dustjs) {
+        into 'web/js'
+      }
+      from ('bower') {
+        into 'web/js'
+      }
+      from (bowerInstall) {
```

Comment:
see line 141. `from (bowerInstall) {`

creates an implicit dependence on bowerInstall from the distribution tasks.